srcs/python/kungfu/tensorflow/policy/policy.py

```diff
@@ -0,0 +1,12 @@
+class Policy():
```

Comment:
should this file be put in
`srcs/python/kungfu/policy/` 
instead of 
`srcs/python/kungfu/tensorflow/policy/`

It is supposed to be the base for both TF and PyTorch.